build.gradle.kts

```diff
@@ -300,14 +301,11 @@ tasks.withType<KotlinCompile> {
 
 	kotlinOptions {
 
-		jvmTarget = "1.8"
+		jvmTarget = javaLangVersion.toString()
```

Comment:
Cool! Looks look like kotlinc can target bytecode specs after 8 now. I didn't know that.